sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # Release History
 
-## 5.3.0-beta.2 (Unreleased)
+## 5.3.0-beta.2 (2020-09-29)
+
+### Changes
+
+#### New Features
+
+- The `EventData` representation has been extended with the ability to treat the `Body` as `BinaryData`, providing support for a variety of data transformations and `ObjectSerializer` integration.
+
+- `EventData` has been integrated with the new Schema Registry service, via use of the `ObjectSerializer` with `BinaryData`.
+
+**Note:** The Schema Registry feature is also a new addition to the service and may not yet be available in all regions or Azure clouds.
```

Comment:
I believe the Schema Registry is being marketed as a separate service, even though it is in the EventHubs realm. So, I don't know if calling it a 'feature' is right(?) The high-level summary of it is:
> Azure Schema Registry is a hosted schema repository service provided by Azure Event Hubs, designed to simplify schema management and data governance. 

So, maybe changing the wordage of this note would help it be more accurate.

Reply:
According to the preview announcements, they're marketing it as a part of Event Hubs:

- [Public Preview of the Azure Schema Registry in Azure Event Hubs](https://techcommunity.microsoft.com/t5/messaging-on-azure/public-preview-of-the-azure-schema-registry-in-azure-event-hubs/ba-p/1699878)
- [Azure Schema Registry in Event Hubs (Preview)](https://docs.microsoft.com/en-us/azure/event-hubs/schema-registry-overview)

Reply:
Gave it a bit of wordsmithing to incorporate your suggestion, but kept the strong association with Event Hubs.